robotbase/src/main/java/frc/robot/subsystems/CameraManager.java

```diff
@@ -71,28 +73,26 @@ public boolean exists() {
      */
     public static poseEstimate averageOutEstimates(poseEstimate... estimates) {
       if (estimates.length == 1) return estimates[0];
-      Pose3d averagedPose = Pose3d.kZero;
+      Translation3d aveTranslation = new Translation3d();
```

Comment:
Good rule of thumb, don't abbreviate or do it as little as possible. You'll thank yourself in 8 months.